addons/hr_holidays/static/src/components/day_selection/day_selection.js

```diff
@@ -0,0 +1,41 @@
+import { registry } from "@web/core/registry";
+import { SelectionField, selectionField } from "@web/views/fields/selection/selection_field";
+
+function getMonthField(dayField){
+    const array = dayField.split("_")
+    array.pop()
+    if (array[array.length - 1] === 'month'){
```

Comment:
you could also let `)` breathe a bit

```suggestion
    if (array[array.length - 1] === 'month') {
```